server/src/cli.rs

```diff
@@ -174,6 +174,10 @@ pub struct EdgeArgs {
     /// Token header to use for both edge authorization and communication with the upstream server.
     #[clap(long, env, global = true, default_value = "Authorization")]
     pub token_header: TokenHeader,
+
+    /// If set to true, Edge starts in open mode. Open mode means that Edge will accept tokens outside of the scope of the startup tokens
+    #[clap(short, long, env, default_value_t = true)]
```

Comment:
Shouldn't we default to false here?

Reply:
Good point, addressed in https://github.com/Unleash/unleash-edge/pull/474/commits/e31c1ce63377e5ca945c63f88b70dbcdf156ddde